fvm/systemcontracts/system_contracts.go

```diff
@@ -176,40 +182,44 @@ func init() {
 	// Main Flow network
 	// All system contracts are deployed to the account of the staking contract
 	mainnet := map[string]flow.Address{
-		ContractNameEpoch:             stakingContractAddressMainnet,
-		ContractNameClusterQC:         stakingContractAddressMainnet,
-		ContractNameDKG:               stakingContractAddressMainnet,
-		ContractNameNodeVersionBeacon: flow.Mainnet.Chain().ServiceAddress(),
+		ContractNameEpoch:               stakingContractAddressMainnet,
+		ContractNameClusterQC:           stakingContractAddressMainnet,
+		ContractNameDKG:                 stakingContractAddressMainnet,
+		ContractNameNodeVersionBeacon:   flow.Mainnet.Chain().ServiceAddress(),
```

Comment:
Consider refactoring the repeated chain expressions into a common variable/"constant"